src/app/forms/forms.component.ts

```diff
@@ -0,0 +1,82 @@
+import {CdkPortalOutlet, DomPortal} from '@angular/cdk/portal';
+import {
+  AfterViewInit,
+  Component,
+  ViewChild,
+  ElementRef, OnInit
+} from '@angular/core';
+import {CdkDragDrop, copyArrayItem, moveItemInArray} from "@angular/cdk/drag-drop";
+import {
+  getStylesBy
+} from "../store/styles.reducer";
+import {FormGroup} from "@angular/forms";
+import {select, Store} from "@ngrx/store";
+import {Observable} from "rxjs";
+import {enumTOArray} from "../store/helper";
+import {ElementType} from "../store/interfaces";
+
+@Component({
+  selector: 'app-forms',
+  templateUrl: './forms.component.html',
+  styleUrls: ['./forms.component.scss']
+})
+export class FormsComponent implements AfterViewInit, OnInit {
+  @ViewChild('accordionContainer', {read: CdkPortalOutlet}) virtualPortalOutlet1: CdkPortalOutlet;
+  @ViewChild('builderContainer', {read: CdkPortalOutlet}) virtualPortalOutlet2: CdkPortalOutlet;
+  @ViewChild('elementsContainer', {read: CdkPortalOutlet}) virtualPortalOutlet3: CdkPortalOutlet;
```

Comment:
use self-descriptive names
```suggestion
  @ViewChild('accordionContainer', {read: CdkPortalOutlet}) virtualPortalOutlet1: CdkPortalOutlet;
  @ViewChild('builderContainer', {read: CdkPortalOutlet}) virtualPortalOutlet2: CdkPortalOutlet;
  @ViewChild('elementsContainer', {read: CdkPortalOutlet}) virtualPortalOutlet3: CdkPortalOutlet;
```